### ISSUE TITLE
Branding and link updates

### ISSUE THREAD

**Issue by ajrobbins** [WikiWatershed/model-my-watershed#2534]
Request from Stroud:

Entry panel for Get Started - 
- Just after the WikiWatershed logo, add the following statement and Stroud logo "An initiative of:"(see PPT slide 1)
- move "Supported by:" and WPF, NSF, SWRC logos down
- If space is available, think about adding hyperlinked text "Meet the Development TEAM" (https://wikiwatershed.org/about/)

Top black bar/row on right corner
- add small stroud logo like previous versions (you should already have the old white logo)  
- "About" link on right corner lands on Wikiwatershed.org entry page, not the about page...please change to https://wikiwatershed.org/about/
- Add a Hyperlinked word "Help" that goes to https://wikiwatershed.org/help/

See reference images here:
[MMWBrandingNeeds.pdf](https://github.com/WikiWatershed/model-my-watershed/files/1514462/MMWBrandingNeeds.pdf)

Need design consult from @jfrankl

**Comment by ajrobbins** [comment]
@jfrankl can you provide the branding designs here?

**Comment by ajrobbins** [comment]
Still blocked while I quickly check with Stroud for signoff on the above designs.

**Comment by ajrobbins** [comment]
Feedback from Stroud:
> Change the about text to:
>"Model My Watershed is part of the WikiWatershed initiative of Stroud Water Research Center. WikiWatershed is a web toolkit designed to support citizens, conservation practitioners, municipal decision-makers, researchers, educators, and students to collaboratively advance knowledge and stewardship of fresh water. 

>Read more about Model My Watershed
>Meet the development team"

Stroud also would like us to integrate the MMW version number in the About modal, with a link to https://wikiwatershed.org/tag/releases/ or https://github.com/WikiWatershed/model-my-watershed/releases. This is important because the model results are dependent on which version we're on, due to dataset and other improvements. @jfrankl can you suggest styling for a hyperlinked version #, and which page would be most helpful to link to?

**Comment by rajadain** [comment]
Should the same guidelines be followed for BiG-CZ? https://portal.bigcz.org/

**Comment by ajrobbins** [comment]
In theory yes, but in practice we are not on contract for that project anymore. Also, a lot of that functionality will be folded into MMW for Monitor my Watershed anyways.

**Comment by rajadain** [comment]
A lot of that code is rather tightly coupled and there might be some work extricating that to not be affected. Will do the minimum necessary.

**Comment by ajrobbins** [comment]
If it's easier to apply to both projects we can do that, but the description and funders will be different for BCZ.

**Comment by rajadain** [comment]
I think the version number is a big enough lift to be its own thing, so I made #2591 for it. Is this unblocked otherwise?

**Comment by ajrobbins** [comment]
Yep!

**Comment by jfrankl** [comment]
How about something like this for the version number?

![image](https://user-images.githubusercontent.com/1809908/34581506-a150e300-f15e-11e7-8aaf-af0cf6759fb4.png)

My intuition is that it should link to the release notes for that specific release on the wikiwatershed page ([example](https://wikiwatershed.org/model-watershed-release-1-21/)). The user wants to learn about the latest updates to MMW. It seems like the language on the wikiwatershed release notes is easier to read and include screenshots, two features that I think will be better for general users of the application.

**Comment by rajadain** [comment]
Should Stroud be in the "With major funding from" section? They currently are in the "Supported by" section:

![image](https://user-images.githubusercontent.com/1430060/34585155-97cc25ca-f16c-11e7-8fd3-71f163610a36.png)

Also, do we lose the WikiWatershed logo completely?
